Source/Core/VideoBackends/D3D/D3DGfx.cpp

```diff
@@ -261,8 +261,11 @@ void Gfx::UnbindTexture(const AbstractTexture* texture)
     D3D::stateman->ApplyTextures();
 }
 
-void Gfx::Flush()
+void Gfx::Flush(bool softFlush)
```

Comment:
Should be `soft_flush` according to guidelines. Same for other uses.

Reply:
The pains of jumping around across projects with different coding styles...

Reply:
Even better might be to make an e.g. `enum class FlushType` for this parameter. `bool`-typed parameters like this make hard to read code at the call-site.

Reply:
Good suggestion.